src/studio/src/designer/frontend/packages/schema-editor/package.json

```diff
@@ -8,10 +8,12 @@
   "jsnext:main": "dist/index.es.js",
   "types": "dist/index.d.ts",
   "peerDependencies": {
+    "@altinn/altinn-design-system": "^0.7.1",
     "@material-ui/core": "^4.12.4",
     "@material-ui/icons": "^4.11.3",
     "@material-ui/lab": "^4.0.0-alpha.61",
     "@reduxjs/toolkit": "^1.8.3",
+    "altinn-designsystem": "^2.7.12",
```

Comment:
I don't think we need this package here, this is the old design system that we will be phasing out.